client/my-sites/email/email-providers-comparison/style.scss

```diff
@@ -194,7 +234,7 @@
 
 			@include breakpoint-deprecated( '>1040px' ) {
```

Comment:
I've missed this breakpoint conversion. I will take care of it on the next 20% allocated time for improvements